platformics/graphql_api/setup.py

```diff
@@ -50,7 +50,7 @@ def get_graphql_name(self, obj: HasGraphQLName) -> str:
         return super().get_graphql_name(obj)
 
 
-def get_app(settings: APISettings, schema: strawberry.Schema, db_module: typing.Any) -> FastAPI:
+def get_app(settings: APISettings, schema: strawberry.Schema, db_module: typing.Any, dependencies: typing.Optional[typing.Sequence[Depends]] = []) -> FastAPI:
```

Comment:
Just out of curiosity, what are these global dependencies? Would they be any better represented as middlewares?

Reply:
basically to check auth tokens. I first wrote this as middleware but it was suggested to me that a dependency might be for that.